server/cmwell-ws/app/controllers/InputHandler.scala

```diff
@@ -323,6 +327,8 @@ class InputHandler @Inject() (ingestPushback: IngestPushback,
               Future(Ok(Json.obj("success" -> true, "dry-run" -> true)))
             else {
 
+              val isPriorityWrite = req.getQueryString("priority").isDefined
```

Comment:
No point in checking variable again.
memoize as a lazy val (or regular eager val if we always perform this check) at the top of `handlePost`